addons/web/static/src/views/fields/placeholder_image/placeholder_image_field.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <templates id="template" xml:space="preserve">
+    <t t-name="web.PlaceholderImageField" t-inherit="web.ImageField" t-inherit-mode="primary">
```

Comment:
Not a fan of the "PlaceholderImageField".

If this is something we want mainly for partners / users, maybe "ContactImageField" ?